### PR TITLE
[v2] Node Client `StoreChunks` method

### PR DESCRIPTION
**Pull request by ian-shim**
## Why are these changes needed?
Initial implementation of v2 node client. It only implements `StoreChunks` method for now. 

## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- [ ] I've checked the new test coverage and the coverage percentage didn't drop.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by ian-shim** [comment]
To be merged on top of https://github.com/Layr-Labs/eigenda/pull/851

File: api/clients/node_client_v2_test.go [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package clients_test
+
```
Comment (cody-littley) [inline api/clients/node_client_v2_test.go:2]: I think you forgot something here. 😄

Comment (ian-shim) [reply api/clients/node_client_v2_test.go]: 🤭 an attempt has been made

**Comment by ian-shim** [comment]
To be rebased on top of https://github.com/Layr-Labs/eigenda/pull/864

**Review by cody-littley — APPROVED** [review]

**Pull request merged**